native-apps/unity_ii_universallink/unity_project/Assets/Scripts/TestICPAgent.cs

```diff
@@ -0,0 +1,75 @@
+using UnityEngine;
+using UnityEngine.UI;
+using EdjCase.ICP.Agent.Agents;
+using EdjCase.ICP.Agent.Identities;
+using EdjCase.ICP.Candid.Models;
+
+namespace IC.GameKit
+{
+    public class TestICPAgent : MonoBehaviour
+    {
+        public string greetFrontend = "https://nytzv-wqaaa-aaaan-qixdq-cai.icp0.io/";
+        public string greetBackendCanister = "n7s7b-3iaaa-aaaan-qixda-cai";
+
+        Text mMyPrincipalText = null;
+        Button mGreetButton = null;
+        Ed25519Identity mEd25519Identity = null;
+        DelegationIdentity mDelegationIdentity = null;
+
+        public Ed25519Identity TestIdentity { get { return mEd25519Identity; } }
```

Comment:
Should this really be `public`?

Reply:
Fixed at https://github.com/dfinity/examples/pull/734/commits/63209beee1eacf11c9c3af23984913fe87e8110c